tests/test_ht_spkitable.c

```diff
@@ -1,391 +1,450 @@
 #include <stdio.h>
 #include <string.h>
 #include <time.h>
 #include <stdlib.h>
 
-static struct spki_record *create_record(int ASN, int ski_offset, int spki_offset, struct rtr_socket *socket);
-
-static bool compare_spki_records(struct spki_record *r1, struct spki_record *r2)
+/* test_ht_4 */
+#define NUM_TABLE_X		(50)
+#define NUM_TABLE_Y		(50)
+/* test_ht_6 */
+#define NUM_SKIS		(10)
+#define NUM_SKIS_RECORDS	(10)
+/* test_ht_7 */
+#define NUM_OF_SKI		(200)
+#define NUM_OF_RECORDS		(2000)
+
+static struct spki_record *create_record(int ASN,
+					 int ski_offset,
+					 int spki_offset,
+					 struct rtr_socket *socket);
+/**
+ * @brief Compare SPKI records for equality
+ *
+ * @return true if r1 == r2, false otherwise
+  */
+static bool compare_spki_records(struct spki_record *r1,
+				 struct spki_record *r2)
 {
-    if(r1->asn != r2->asn) {
-        return false;
-    }
-    if(r1->socket != r2->socket) {
-        return false;
-    }
-    if(memcmp(r1->ski, r2->ski, SKI_SIZE) != 0) {
-        return false;
-    }
-    if(memcmp(r1->spki, r2->spki, SPKI_SIZE) != 0) {
-        return false;
-    }
-    return true;
+	if (r1->asn != r2->asn)
+		return false;
+	if (r1->socket != r2->socket)
+		return false;
+	if (memcmp(r1->ski, r2->ski, SKI_SIZE) != 0)
+		return false;
+	if (memcmp(r1->spki, r2->spki, SPKI_SIZE) != 0)
+		return false;
+
+	return true;
 }
 
-static struct spki_record *create_record(int ASN, int ski_offset, int spki_offset, struct rtr_socket *socket)
+/**
+ * @brief Create a SPKI record
+ *
+ * @return new SPKI record
+ */
+static struct spki_record *create_record(int ASN,
+					 int ski_offset,
+					 int spki_offset,
+					 struct rtr_socket *socket)
 {
-    struct spki_record *record = malloc(sizeof(struct spki_record));
-    memset(record, 0, sizeof(*record));
+	struct spki_record *record = malloc(sizeof(struct spki_record));
+	uint32_t i;
+
+	memset(record, 0, sizeof(*record));
+	record->asn = ASN;
 
-    record->asn = ASN;
-    uint32_t i;
+	for (i = 0; i < sizeof(record->ski) / sizeof(u_int32_t); i++)
+		((u_int32_t *)record->ski)[i] = i + ski_offset;
 
-    for(i = 0; i < sizeof(record->ski)/sizeof(u_int32_t); i++){
-        ((u_int32_t*)record->ski)[i] = i + ski_offset;
-    }
+	for (i = 0; i < sizeof(record->spki) / sizeof(u_int32_t); i++)
+		((u_int32_t *)record->spki)[i] = i + spki_offset;
 
-    for(i = 0; i < sizeof(record->spki)/sizeof(u_int32_t); i++){
-        ((u_int32_t*)record->spki)[i] = i + spki_offset;
-    }
-    record->socket = socket;
-    return record;
+	record->socket = socket;
+	return record;
 }
 
 /**
- * @brief test_ht_1
- * Test if the spki_table_src_remove function is working correctly by adding
- * spki_record associated with different rtr_socket. Then call spki_table_src_remove
- * with one of the sockets as argument and validate that all records associated with different
- * sockets are still there.
+ * @brief Helper shortcut, to assert insert of table entry
```

Comment:
Could be helpful to description what does it assert exacty, when will the assert fail?